usr/share/rear/skel/default/etc/scripts/run-sshd

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 # check inittab for "ssh:23:respawn:/bin/sshd -D"
 if $(grep -q ^ssh: /etc/inittab) ; then
+    if ! ls /etc/ssh/*key* &>/dev/null ; then
```

Comment:
To be safe, if just a public key file exists (`*_key.pub`) but no private key, change this line to:
```
if ! ls /etc/ssh/*_key &>/dev/null ; then
```

In my view, even better: Just check if an "rsa" key exists and generate one, if not. It doesn't hurt and helps to enable access in case of mis-configurations:
```
if ! [[ -f /etc/ssh/ssh_host_rsa_key ]]; then
```

Reply:
Done.